packages/grid/_modules_/grid/components/columnHeaders/GridColumnHeaderItem.tsx

```diff
@@ -149,6 +153,10 @@ export const GridColumnHeaderItem = ({
     }
   });
 
+  const isFirstTabbable =
```

Comment:
I'm not sure that the first column header should be tabbale. I was left with the impression that the first cell should be tabble.

Reply:
That's the thing, IMO, the first tabbable control of a sortable grid should be the first sortable column header
https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html

So in our case, I put the first column so we could argue that if none of the columns have an action then it should be the first cell of the first row

Reply:
Actually, if we use that logic then the first tabbable cell should be either the column header that is sortable or the one that has a column menu as it is also a interaction.